code/datums/mood_events/generic_negative_events.dm

```diff
@@ -142,6 +142,16 @@
 	description = "<span class='boldwarning'>I recently saw my own corpse...</span>\n"
 	mood_change = -6
 
+/datum/mood_event/someone_died
+	description = "<span class='warning'>I saw someone die, and it has affected me deeply.</span>\n"
+	mood_change = -6
+	timeout = 10 MINUTES
```

Comment:
There's an argument to be made about other moodlets expiring way too fast but that needs to change first, with a 10 minutes duration this is going to require constant attention to counter and if you're playing medical or shit is hitting the fan, you're going to have to either be insane or constantly drunk

Reply:
I feel like I can reasonably make a PR soon that buffs times for moodlets.